docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
+title: Browser Support
+permalink: /documentation/using-testcafe/common-concepts/browsers/browser-support.html
+---
+# Browser Support
+
+* [Officially Supported Browsers](#officially-supported-browsers)
+* [Locally Installed Browsers](#locally-installed-browsers)
+* [Portable Browsers](#portable-browsers)
+* [Browsers on Remote Devices](#browsers-on-remote-devices)
+* [Browsers in Cloud Testing Services](#browsers-in-cloud-testing-services)
+* [Nonconventional Browsers](#nonconventional-browsers)
+
+## Officially Supported Browsers
+
+While TestCafe is designed to support most modern browsers, there are a limited number
+of *officially supported* browsers against which TestCafe is actively tested.
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (9+)
+* Microsoft Edge
+* Mozilla Firefox
+* Safari
+* Android browser (the latest available on SauceLabs)
```

Comment:
I think we shouldn't mention SauceLabs here, also `the latest available on SauceLabs` means nothing without the link to supported platforms,